consensus/polybft/fsm_test.go

```diff
@@ -210,7 +205,7 @@ func TestFSM_BuildProposal_WithCommitEpochTxGood(t *testing.T) {
 	stateBlock := createDummyStateBlock(parentBlockNumber+1, parent.Hash, extra)
 
 	mBlockBuilder := newBlockBuilderMock(stateBlock)
-	mBlockBuilder.On("WriteTx", mock.Anything).Return(error(nil)).Twice()
+	mBlockBuilder.On("WriteTx", mock.Anything).Return(error(nil)).Repeatability = 3
```

Comment:
```suggestion
	mBlockBuilder.On("WriteTx", mock.Anything).Return(error(nil)).Times(3)
```

Just for consistency, because .Times() is used everywhere else in the codebase

Reply:
Actually, in most of the places it uses `.Once()` or `.Twice()`, but I agree that `.Times()` is better than `.Repeatability`.

Reply:
Times is used when > 2